src/maintainers.sh

```diff
@@ -160,9 +160,23 @@ function print_files_authors()
   local printed_authors_separator=false
 
   for file in "${files[@]}"; do
-    authors=$(grep --only-matching --extended-regexp 'MODULE_AUTHOR *\(.*\)' "$file" |
-      sed --regexp-extended 's/(MODULE_AUTHOR *\( *\"|\" *\))//g' |
-      sed --expression ':a' --expression 'N' --expression '$!ba' --expression 's/\n/, /g')
+    # Get every content inside the MODULE_AUTHOR's parenthesis (even if it is in multiple lines)
+    # Here grep matches 'MODULE_AUTHOR (' followed by a sequence of non-')' characters
+    # But it only matches the characters to the right of \K thanks to its look-behind nature
+    authors=$(grep --null-data --perl-regexp --only-matching 'MODULE_AUTHOR *\(\K[^)]*' "$file" |
+      tr --delete '\0')
+
+    # Now we start trimming the content from the first grep
+    # Delete possible hard written '\n' from authors names
+    # Delete the backslash '\' with any possible adjacent white spaces
+    # Substitute possible white spaces at the beginning of the lines for a single one
```

Comment:
If possible, don't break it in phrases and say it directly in the imperative way to make it more succinct. For example,

```bash
# Trim contents from first grep deleting `\n` and `\` with any adjacent
# white spaces, and substitute leading white spaces to a single one
```

Again, this is a rough draft, and just a suggestion